ignition/README.md

```diff
@@ -1,23 +1,25 @@
 # The configuration of the Ignition network
 
 ## Chain config
-- The `ChainId` is `0`.
-- The initial `privileged_address` which can perform the network upgrade: b62c6f41c4c380d78ae67a8b432a4a41e2465383f8cc9869b67106835685c388
-- The public address of the authority node that produces blocks: e8df6d432b2584a3ea9d0badf297c3b525bae71b577d22c1ccc12519adb64d92
+- The `ChainId` is `9889`.
+- The initial `privileged_address` which can perform the network upgrade: `eb42c979046d091f4f524e1efd8d4ffad92629b9a506b268a511861b0dbf2366`
+- The genesis public address of the authority node that produces blocks: `da6b021bf570f126869591f2009a95efb826f69272987db699fdea1de813697e`
 - The block gas limit is `30000000`.
+- Genesis state transition version is set to `10` - corresponding release is `fuel-core 0.35.0`.
```

Comment:
The `0.35.0` release uses version 10 of the state transition function.

<img width="584" alt="image" src="https://github.com/user-attachments/assets/d224f48d-91dd-4bc3-a538-fce03fe40901">